staff/carlos-calvo/skylabcoders-html-semantics/home.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HOME SKYLAB CODERS</title>
+</head>
+<body>
+    <!--Franja superior blanca con log y con el menu-->
+    <header id="franja superior">
+        <h1>Skylabcoders
+            <a href="https://www.skylabcoders.com/en">
+            <img src = "https://www.skylabcoders.com/images/403/default.png"></img></a>
+        </h1>
+        <input placeholder="" type="search" class="input">
+        <ul class="TreeView Menu menu" data-menu-state="collapsed">
+            <li class="selected active menu-entry-skylab-home_page"><strong><a href="https://www.skylabcoders.com/en">Home</a></strong></li> <li class="menu-entry-skylab-courses_page"><a href="https://www.skylabcoders.com/en/courses_3574">Courses</a><ul data-menu-state="collapsed"><li><a href="https://www.skylabcoders.com/en/full-time-bootcamp_5278">Full time - Bootcamp</a></li><li class="menu-entry-skylab-courses_page-part_time"><a href="https://www.skylabcoders.com/en/part-time-course_25971">Part time course</a></li></ul></li><li class="menu-entry-skylab-who_we_are_page"><a href="https://www.skylabcoders.com/en/faculty_3803">Faculty</a></li><li class="menu-entry-skylab-why_code_page"><a href="https://www.skylabcoders.com/en/why-code-_4139">Why code?</a></li><li class="menu-entry-skylab-companies_page"><a href="https://www.skylabcoders.com/en/companies_4434">Companies</a></li><li class="menu-entry-skylab-community_page"><a href="https://www.skylabcoders.com/en/community_4573">Community</a></li><li class="menu-entry-skylab-contact_page"><a href="https://www.skylabcoders.com/en/contact_4078">Contact</a></li></ul>
+        <button type="button" class="lang_button" id="clientElement2">en</button>
+        <button type="button" class="burger_button active" id="clientElement3">
+            <img src="/706.png" alt="SkyLab Coders Academy Menu">
+        </button>
+    </header>
+    <main>
+        <section>
+            <h2>Learn&nbsp;to code<br>in 11 weeks<br>and find&nbsp;a job</h2>
+            <p>We offer&nbsp;<strong>coding&nbsp;courses</strong>&nbsp;where you learn web and mobile applications development.</p>
+            <p>We help you to learn Github + HTML + CSS + Javascript + TDD + ReactJS + NodeJS + MongoDB to&nbsp;<strong>find a job</strong>.</p>
+            <img src="/fitxer-23068_23068.png" alt="Best coding bootcamp in Barcelona">
+        </section>
+        <section>
+            <h3 class="heading">In Skylab we do not just teach you programming, we make you programmer</h3>
+            <p><span class="highlighted_data"><span class="data">96</span>%&nbsp;</span><br><strong>get a web<br>development job</strong></p>
```

Comment:
WARN! `span` is NOT semantic. the challenge is just to filter semantic content, and remove the opposite.